Azure/azure-cli
# [AppService] az webapp config ssl bind: handle if webapp and appservice plan in different rg. Also reference text updates

**Description**
Fixes #16642. If webapp and appservice plan is in different rg because webapp got moved, `az webapp config ssl bind` will fail with "cert not found" 

Also added in two reference text updates

**Testing Guide**
- Create a webapp 
- Move webapp to a different resource group than the appservice plan
- `az webapp config ssl import` command to import SSL cert to webapp that has an appservice plan in diff resource group
- `az webapp config ssl bind` - this command will fail with "certificate not found" error if webapp and appservice plan in diff resource groups

---

This checklist is used to make sure that common guidelines for a pull request are followed.
